src/main/java/org/folio/service/inventory/InventoryItemManager.java

```diff
@@ -390,7 +390,7 @@ private Future<JsonObject> buildBaseItemRecordJsonObject(CompositePoLine compPOL
   public Future<String> createBindItem(CompositePoLine compPOL, String holdingId, BindItem bindItem, RequestContext locationContext) {
     JsonObject item = new JsonObject()
       .put(ITEM_HOLDINGS_RECORD_ID, holdingId)
-      .put(ITEM_STATUS, new JsonObject().put(ITEM_STATUS_NAME, ReceivedItem.ItemStatus.ON_ORDER.value()))
+      .put(ITEM_STATUS, new JsonObject().put(ITEM_STATUS_NAME, ReceivedItem.ItemStatus.IN_PROCESS.value()))
```

Comment:
per this AC:
![image](https://github.com/folio-org/mod-orders/assets/25097693/42364f7b-cd22-4957-b27e-3f2526b0a6ee)